app/models/text_condition.rb

```diff
@@ -0,0 +1,14 @@
+class TextCondition < OfferCondition
```

Comment:
Is there a better name for this general purpose kind of condition?

Reply:
For standard conditions should we implement an enum field (under `detail`)?

Reply:
i think the name is fine